app/src/main/java/com/github/houseorganizer/houseorganizer/Task.java

```diff
@@ -108,10 +108,14 @@ public SubTask getSubTaskAt(int index) {
         return subtasks.get(index);
     }
 
-    public boolean hasDueDate() {
-        return dueDate != LocalDateTime.MIN;
+    public List<SubTask> getSubTasks() {
+        return subtasks;
```

Comment:
maybe return a clone if its not used to modify the list

Reply:
Should work now. In any case my classes are not fully immutable, but that can be changed in future sprints.